arts.php

```diff
@@ -160,7 +160,12 @@
         // Prevent the script from timing out
         set_time_limit(0);
 
-        $image      = Art::get_from_source($_SESSION['form']['images'][$image_id], 'album');
+        $show_song_art = AmpConfig::get('show_song_art') ?: false;
```

Comment:
The ternary operatioin `?:` isn't necessary and also isn't fool proof.  If you look at the function definition ` AmpConfig::get('show_song_art')` it has a second parameter  which can set a return value.

Just a reminder for the next PR you do.

Reply:
these are changed into different operators based on need now, the false isn't required in the way it's written